Description: fix paths to match Debian FHS
 As lintian says:
  The manual page references a directory that is specified
  in the FSSTND but not in the FHS which is used by Debian.
  This can be an indicator of a mismatch of the location of
  files as installed for Debian and as described by the man
  page.
Origin: vendor
Forwarded: not-needed
Author: Jonathan Yu <jawnsy@cpan.org>
Reviewed-by: Nicholas Bamber <nicholas@periapt.co.uk>
Reviewed-by: gregor herrmann <gregoa@debian.org>
Last-Update: 2014-07-24

--- a/lib/Inline/C/Cookbook.pod
+++ b/lib/Inline/C/Cookbook.pod
@@ -175,8 +175,8 @@
 
 Running this script produces:
 
-    > perl vowels.pl /usr/dict/words
-    The letters in /usr/dict/words are 37.5% vowels.
+    > perl vowels.pl /usr/share/dict/words
+    The letters in /usr/share/dict/words are 37.5% vowels.
 
 It is very important to note that the examples in this cookbook use C<char *>
 to mean a string. Internally Perl has various mechanisms to deal with strings
